app/projects/[id]/page.tsx

```diff
@@ -45,24 +45,17 @@ export default function ProjectDetailPage() {
             {findProject.name}
           </h1>
           <div className="flex flex-wrap items-center justify-start gap-5">
-            {githubLink !== "none" && (
-              <NextLink
-                href={findProject.links.github}
-                target="_blank"
-                rel="noreferrer"
-              >
-                <div className="flex items-center gap-2">
-                  <NextImage
-                    src={GithubVector}
-                    alt="bg"
-                    width={20}
-                    height={20}
-                  />
-                  <p>Github</p>
-                </div>
-              </NextLink>
-            )}
-            {websiteLink !== "none" && (
+            <NextLink
```

Comment:
why did you remove the conditional checking for github?

Reply:
because every project have github